taichi/codegen/codegen_llvm.cpp

```diff
@@ -1771,7 +1771,9 @@ void CodeGenLLVM::create_offload_struct_for(OffloadedStmt *stmt, bool spmd) {
   // When looping over bit_arrays, we generate struct for on bit array's parent
   // node (usually "dense") instead of itself for higher performance. Also, note
   // that the loop must be bit_vectorized.
```

Comment:
```suggestion
  // When looping over bit_arrays and bit_structs, we generate struct for on their parent
  // node (usually "dense") instead of itself for higher performance. Also, note
  // that the loop must be bit_vectorized for bit_arrays.
```